src/test/java/ancaMarian/tests/tema8/tests/DisabledDynamicButtonsPageTest.java

```diff
@@ -0,0 +1,29 @@
+package ancaMarian.tests.tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.DisabledDynamicButtonsPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class DisabledDynamicButtonsPageTest extends BaseTest {
+
+    @Test(groups = {"mobile", "desktop"})
+    public void dynamicButtonsPageTest() {
+        driver.get("https://testpages.herokuapp.com/styled/dynamic-buttons-disabled.html");
+        DisabledDynamicButtonsPage disabledDynamicButtonsPage = new DisabledDynamicButtonsPage(driver);
+        disabledDynamicButtonsPage.waitForStartButtonToBeClickable();
+        disabledDynamicButtonsPage.clickOnStartButton();
+        disabledDynamicButtonsPage.waitForOneButtonToBeClickable();
+        disabledDynamicButtonsPage.clickOnOneButton();
+        disabledDynamicButtonsPage.waitForWaitMessageToDisappear();
+        disabledDynamicButtonsPage.clickOnTwoButton();
+        disabledDynamicButtonsPage.waitForWaitMessageToDisappear();
+        disabledDynamicButtonsPage.clickOnThreeButton();
+        if (disabledDynamicButtonsPage.waitForConfirmationMessageToAppear()) {
```

Comment:
Replace this if with the assertion of the expected message after all buttons were clicked. Similar with the one from DynamicButtonsSimpleTest class. Thanks!